build/integrationtests/startup.sh

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env bash
 
-source $HOME/.setup_sal_env.sh
-
-$RUN_ARG &
+source ${HOME}/.setup_sal_env.sh
```

Comment:
I am not sure what Michael wants with these images but I had the impression he was controlling what to run using `RUN_ARG`. Unless I am mistaken, you can just leave this startup file as it is.

Reply:
Done.  Added a dummy value for RUN_ARG back to the Dockerfile.  That'll get updated once we figure out what it should be.